src/sentry/grouping/strategies/newstyle.py

```diff
@@ -323,9 +303,6 @@ def get_function_component(
                 hint="ignored because sourcemap used and context line available",
             )
 
-    if function_component.values and context["hierarchical_grouping"]:
-        function_component.update(tree_label={"function": function_component.values[0]})
```

Comment:
Undoes the changes from this other PR:
https://github.com/getsentry/sentry/commit/ea45a51265fe0f95ad2896d30e7ba4b3df92890e#diff-765101e1faf45dddc42ee634ea872031228b3072cd6a28d41c533c3aba7414d0R230